packages/frontend/web-ui/src/game/hooks/useGetMessageFinishedGame.ts

```diff
@@ -0,0 +1,52 @@
+import { models as apiModels } from '@cornie-js/api-models';
+import { useEffect, useState } from 'react';
+
+import { useGetUser, UseGetUserResult } from '../../user/hooks/useGetUser';
+
+export interface UseGetMessageFinishedGameResult {
+  messageFinishedGame: string | undefined;
+}
+
+export const useGetMessageFinishedGame = (
+  game: apiModels.GameV1 | undefined,
+): UseGetMessageFinishedGameResult => {
+  const [messageEndGame, setMessageEndGame] = useState<string | undefined>(
```

Comment:
```suggestion
  const [endGameMessage, setEndGameMessage] = useState<string | undefined>(
```

Reply:
I've changed these names to `endGameWinner` and `setEndGameWinner` to provide a `apiModels.UserV1` user.